assets/js/components/notifications/IdeaHubPromptBannerNotification.js

```diff
@@ -59,6 +59,12 @@ export default function IdeaHubPromptBannerNotification() {
 		select( MODULES_IDEA_HUB )?.getAdminReauthURL()
 	);
 
+	const ideaHubSupportURL = useSelect( ( select ) => {
+		return select( CORE_SITE ).getDocumentationLinkURL(
+			'using-site-kit/idea-hub'
```

Comment:
This slug should just be `idea-hub`
```suggestion
			'idea-hub'
```

Reply:
Oh shoot, thanks! 👍🏻